.github/workflows/ci.yml

```diff
@@ -195,10 +195,8 @@ jobs:
     strategy:
       matrix:
         cluster_version:
-         - 'v1.18.0'
-         - 'v1.19.0'
-         - 'v1.20.0'
-         - 'v1.21.0'
```

Comment:
Have we chosen to drop explicit support for 1.18 ?

I'd like to add that back at least

Reply:
The test was failing due to unsupported kubernetes object api versions in 1.18, that's why I removed it for now.

Reply:
```
unable to recognize "tests/manifests/csr.yaml": no matches for kind "CertificateSigningRequest" in version "certificates.k8s.io/v1"

unable to recognize "tests/manifests/ingress.yaml": no matches for kind "Ingress" in version "networking.k8s.io/v1" 
```

Reply:
Yes, this was due to deprecated version that was spamming the logs. https://github.com/kubernetes/kube-state-metrics#compatibility-matrix 

> Note: The v2.0.0-alpha.2+ and master releases of kube-state-metrics work on Kubernetes v1.17 and v1.18 excluding Ingress or CertificateSigningRequest resource metrics. If you require those metrics and are on an older Kubernetes version, use v2.0.0-alpha.1 or v1.9.8 kube-state-metrics release.

I since found out you can tweak client-go not to log this deprecated spam msgs, so we can add back the support if there are many requests for it. Up to you all!